app/generate/head.tsx

```diff
@@ -0,0 +1,7 @@
+export default function Head() {
+    return (
+      <>
+        <title>Generate !!</title>
+      </>
+    )
+  }
```

Comment:
I think we won't need a head overload, it was cool to test but not mandatory now 👉🏼 🗑️

Reply:
Done 👍